src/main/java/com/amazon/janusgraph/graph/AbstractGraphFactory.java

```diff
@@ -0,0 +1,10 @@
+package com.amazon.janusgraph.graph;
+
+import org.janusgraph.graphdb.database.StandardJanusGraph;
+
+/**
+ * Created by addisonslabaugh on 6/22/17.
+ */
+public interface AbstractGraphFactory {
```

Comment:
Similar though process to the abstract class above. I was essentially trying to abstract away as much of the process as possible. Maybe this is too abstracted.